### DISCUSSION TITLE
Would it make sense to use Waku with Electron.js? Is there even a concept of a server on desktop apps?

### DISCUSSION THREAD

**Discussion by clearly-outsane** [wakujs/waku#288] [Q&A]
Was just wondering if anyone has tried supporting things like RSC and server actions when using React with Electron.js. It doesn't make sense to me but I don't know enough to be certain.

**Comment by dai-shi** [comment]
I think @tom-sherman was trying something in Electron. #110 

I'm not super familiar with Electron, but conceptually using RSC should make sense.
Waku's focus is currently Web, but it would be nice to explore non-Web usage.

**Reply by himself65** [reply]
electron 28 supports pure ESM module. I think it will easy more

**Comment by himself65** [comment]
I had a try and run successful

**Reply by himself65** [reply]
![image](https://github.com/dai-shi/waku/assets/14026360/c64c8c5e-5b54-4609-8986-7110a81f4525)
